shared/notifications/index.js

```diff
@@ -494,5 +503,19 @@ export const getTranslatedUINotifications = (
             },
     },
     ///: END:ONLY_INCLUDE_IF
+    [NOTIFICATION_PETNAMES]: {
+      ...UI_NOTIFICATIONS[NOTIFICATION_PETNAMES],
+      title: t('notificationsPetnamesTitle'),
+      description: [
+        t('notificationsPetnamesDescriptionOne'),
+        t('notificationsPetnamesDescriptionTwo'),
+      ],
+      actionText: t('notificationsPetnamesActionText'),
+      date: UI_NOTIFICATIONS[NOTIFICATION_PETNAMES].date
+        ? new Intl.DateTimeFormat(formattedLocale).format(
+            new Date(UI_NOTIFICATIONS[NOTIFICATION_OPEN_BETA_SNAPS].date),
```

Comment:
If it's a standard and we want this logic anyway, should this be `NOTIFICATION_PETNAMES`?

Reply:
You caught my copy-paste error. Thank you!